src/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -107,6 +107,7 @@
     <Compile Include="$(CommonTestPath)\System\PerfUtils.cs">
       <Link>Common\System\PerfUtils.cs</Link>
     </Compile>
+    <Compile Condition="'$(TargetGroup)'=='netstandard1.7'" Include="netstandard.1.7.Tests.cs" />
```

Comment:
This is fine for now but it won't scale. Eventually we will want to add more tests and at that point we should move the condition to an item group and name the files based on what they are testing.